src/test/scala/com/lightbend/kafkalagexporter/InfluxDBPusherSinkTest.scala

```diff
@@ -32,37 +33,37 @@ class InfluxDBPusherSinkTest extends fixture.FreeSpec with Matchers
     test(Fixture(properties, port))
   }
 
+  def doQuery(url: String, query: String): String = {
+    val database = "kafka_lag_exporter"
+    val cmd = Seq("curl", "-G", s"$url/query", "--data-urlencode", s"db=$database", "--data-urlencode", s"q=$query")
+    val output = cmd.!!
+    return output
+  }
+
   "InfluxDBPusherSinkImpl should" - {
 
     "create database" in { fixture =>
       val sink = InfluxDBPusherSink(new InfluxDBPusherSinkConfig("InfluxDBPusherSink", List("kafka_consumergroup_group_max_lag"), ConfigFactory.parseMap(mapAsJavaMap(fixture.properties))), Map("cluster" -> Map.empty))
-      Thread.sleep(1000)
       val port = fixture.port
       val url = s"http://localhost:$port"
       val query = "SHOW DATABASES"
-      val cmd = Seq("curl", "-G", s"$url/query", "--data-urlencode", s"q=$query")
-      val output = cmd.!!
-      output should include("kafka_lag_exporter")
+
+      eventually (timeout(Span(5, Seconds)), interval(Span(500, Millis))) { doQuery(url, query) should include("kafka_lag_exporter") }
```

Comment:
You don't need to pass in parameters to `eventually`. The configuration brought in by the `IntegrationPatience` trait is sufficient.

Reply:
Sure, this is also updated accordingly.